### PR TITLE
added arm builds

### PR DESCRIPTION
**Pull request by mikhail-klimko**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/arm-builds/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/arm-builds/docs/

**Comment by jesse-codefresh** [comment]
@kostis-codefresh 
can you review and tell me what you think?

**Review by kostis-codefresh — APPROVED** [review]

File: _docs/administration/codefresh-runner.md [hunk 1]
```diff
@@ -1601,6 +1601,150 @@ Override environment variables for `dind-lv-monitor` daemonset if necessary:
 - `KB_USAGE_THRESHOLD` - default 80 (percentage)
 - `INODE_USAGE_THRESHOLD` - default 80 
 
+## ARM Builds
+
+With hybrid runner it's possibe to run native ARM64v8 builds.
+
+>**Note:** Running both amd64 and arm64 images within the same pipeline - it is not possible. We do not support multi-architecture builds. One runtime configuration - one architecture. Considering one pipeline can map only to one runtime, it is possible to run either amd64 or arm64, but not both within a one pipeline
+
+The following scenario is an example of how to set up ARM Runner on existing EKS cluster:
```
Comment (francisco-codefresh) [inline _docs/administration/codefresh-runner.md:1610]: ... how to set up **an** ARM Runner on **an** existing EKS...

**Review by jesse-codefresh — APPROVED** [review]

File: _docs/administration/codefresh-runner.md (hunk 1 above)
Comment (francisco-codefresh) [reply _docs/administration/codefresh-runner.md:1610]: @jesse-codefresh , Some missing articles: **an**

**Pull request merged**